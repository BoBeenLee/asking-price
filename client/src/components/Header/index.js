import React from 'react';
import { Menu, Icon } from 'antd';
import styled from 'react-emotion';

const MenuItem = styled(Menu.Item) `
    display: flex;
    flex-direction: row;
    /* justify-content: center; */
    align-items: center;
    height: 80px;
`;

const Header = () => {
    return (
        <Menu
            mode="horizontal">
            <MenuItem key="home">
                <Icon type="home" />Home
            </MenuItem>
        </Menu>
    );
};

export default Header;