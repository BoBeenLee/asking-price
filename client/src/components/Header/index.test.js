import React from 'react';
import renderer from 'react-test-renderer';

import Header from './';

describe('header', () => {
    it('renders correctly', () => {
        const tree = renderer
            .create(<Header />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});