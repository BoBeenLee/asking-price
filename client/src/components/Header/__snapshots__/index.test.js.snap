// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header renders correctly 1`] = `
<ul
  aria-activedescendant=""
  className="ant-menu ant-menu-light ant-menu-root ant-menu-horizontal"
  onKeyDown={[Function]}
  role="menu"
  style={Object {}}
  tabIndex="0"
>
  <li
    aria-disabled={undefined}
    aria-selected={false}
    className="ant-menu-item css-1a6fjph e1a9n7yf0"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    role="menuitem"
    style={Object {}}
    title={undefined}
  >
    <i
      className="anticon anticon-home"
    />
    Home
  </li>
</ul>
`;
